src/CoinbaseWalletSDK.ts

```diff
@@ -10,12 +10,9 @@ import { WalletSDKRelay } from "./relay/WalletSDKRelay"
 import { WalletSDKRelayEventManager } from "./relay/WalletSDKRelayEventManager"
 import { getFavicon } from "./util"
 
-const LINK_API_URL =
-    process.env.LINK_API_URL! || "https://www.walletlink.org"
+const LINK_API_URL = process.env.LINK_API_URL! || "https://www.walletlink.org"
```

Comment:
unrelated to this PR, why is the `!` necessary? the `||` should tell TS that the type of the var is `string`

Reply:
Good catch. I haven't authored 99% of the current codebase so it's likely much of the code were not caught in code reviews or even lint rules.